.ci/Jenkinsfile

```diff
@@ -70,19 +69,19 @@ pipeline {
      */
     stage('Test') {
       steps {
-        deleteDir()
-        unstash 'source'
+        cleanup()
         dir("${BASE_DIR}"){
-          insideGo(){
-            sh(label: 'Runs the (unit) tests',script: 'mage -debug test ')
+          withMageEnv(){
+            sh(label: 'Runs the (unit) tests',script: 'mage -debug test|tee tests-report.txt')
```

Comment:
ditto

```suggestion
            sh(label: 'Runs the (unit) tests',script: 'mage -debug test 2>&1 | tee tests-report.txt')
```

Reply:
I have tested and there is nothing in the stderr even do some tests fail